node/cli/src/run_as_parachain.rs

```diff
@@ -3,41 +3,39 @@ use cumulus_primitives_core::ParaId;
 use frequency_service::chain_spec;
 use log::info;
 use sc_cli::SubstrateCli;
-use sp_runtime::traits::AccountIdConversion;
 
 pub fn run_as_parachain(cli: Cli) -> sc_service::Result<(), sc_cli::Error> {
 	let runner = cli.create_runner(&cli.run.normalize())?;
+	let collator_options = cli.run.collator_options();
+
 	runner.run_node_until_exit(|config| async move {
+		let hwbench = (!cli.no_hardware_benchmarks)
+			.then_some(config.database.path().map(|database_path| {
+				let _ = std::fs::create_dir_all(database_path);
+				sc_sysinfo::gather_hwbench(Some(database_path))
+			}))
+			.flatten();
+
 		let para_id = chain_spec::Extensions::try_get(&*config.chain_spec)
-			.map(|e| e.para_id)
+			.map(|e: &chain_spec::Extensions| e.para_id)
 			.ok_or("Could not find parachain ID in chain-spec.")?;
-		let id = ParaId::from(para_id);
-
-		let parachain_account =
-			AccountIdConversion::<polkadot_primitives::AccountId>::into_account_truncating(&id);
 
-		info!("Parachain id: {:?}", id);
-		info!("Parachain Account: {}", parachain_account);
```

Comment:
no more parachain account?

Reply:
Yes, I think this might be leftover code since it was only used for logging. The PR description where Parity removed it doesn’t provide a clear explanation.
[Link to the PR](https://github.com/paritytech/polkadot-sdk/pull/3801/files).